apps/prairielearn/elements/pl-string-input/pl-string-input.py

```diff
@@ -227,6 +243,9 @@ def parse(element_html: str, data: pl.QuestionData) -> None:
     if remove_spaces:
         a_sub = "".join(a_sub.split())
 
+    # Always simplify multiline characters (if they still exist)
+    a_sub = re.sub("\r*\n", "\n", a_sub)  # noqa: RUF039
```

Comment:
Here and elsewhere in this PR, I'm not sure why this isn't just the following:

```suggestion
    a_sub = re.sub("\r\n", "\n", a_sub)  # noqa: RUF039
```

I think there are two things to consider here:

- When do we expect multiple `\r` characters in a row? I don't think we ever do?
- Do we gain anything by replacing `\n` with `\n`? I definitely don't think we do.

Reply:
I think the original intent was some form of making this applicable to more general situations? Agree that it's probably a bit extra (I think this was actually written by someone else, but can't quite remember).

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/9438/commits/03241c5ae7ce5928d322b5950e07c35f5baf4489